### ISSUE TITLE
[BUG]: aria-labels missing

### ISSUE THREAD

**Issue by CBID2** [Learn-Build-Teach/deals-for-devs#62]
**Describe the bug**
There are some missing aria-labels, which goes against [WCAG 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**To Reproduce**
N/A

**Expected behavior**
N/A
**Screenshots**
N/A
**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Suggested Solution**
Share you how you plan go about solving this bug. 
1. Improve the aria-label attribute to the following code snippet:
- `<nav class="mb-[79px] flex w-full items-center px-6 md:mb-[153px] md:px-0" aria-label="Global" speechify-initial-font-family="__Raleway_98db86, __Raleway_Fallback_98db86" speechify-initial-font-size="16px" speechify-initial-font-size-modified="true" style="font-size: 23.36px;">`

2. Add a `aria-label` to the following code snippet:

`<input placeholder="E-mail address" class="w-full rounded-md border border-white/[.67] bg-transparent p-3 pl-4 text-white md:h-[70px] md:w-[476px] md:rounded-[16px] md:placeholder:text-white/[.47]" type="email" name="email" speechify-initial-font-family="__Raleway_98db86, __Raleway_Fallback_98db86" speechify-initial-font-size="16px">`

3. Add alt text to Sentry and Xata icons

4. Do accessibility tests 

**Additional context**
